FpsCheck.py

```diff
@@ -469,32 +454,29 @@ def find_invalid_alias_eSLDs(self, check_sets):
             None
         """
         for primary, curr_set in check_sets.items():
-            if not curr_set.ccTLDs:
-                continue
-            for aliased_site in curr_set.ccTLDs:
+            for alias, aliased_sites in curr_set.ccTLDs.items():
```

Comment:
I think `alias` and `aliased_site` should be switched here, and in the uses below. Otherwise they're inconsistent with our other references to the field, and with the error messages below (e.g. "The aliased site `alias`"). In general, the key in the pair is the site being aliased, and the value is a list of aliases for that site so I believe `aliased_site`: [aliases] work better as the variable names.

Reply:
Agreed about the mismatch - I was trying to follow the existing usage on lines 208-218 of the original. I've modified all the usages to use the more accurate naming, and fixed some of the error messages to be more accurate.

Also discovered that we were calling `self.check_list_sites` in a loop, once for each aliased site in the set, when we only really have to call it once.